source/Willow-Bootstrap.package/BootstrapDropdownMenuBuilder.class/instance/labeled..st

```diff
@@ -2,8 +2,8 @@ Configuring
 labeled: aRenderable
 
 	buttonBinding := Binding to:
-		(AsynchronicAnchorWebView
-			labeled: [ :canvas | 
+		(supplier
```

Comment:
Please revert this change, the markup for this components specifically needs a plain anchor.